content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -101,17 +101,17 @@ Pick a control plane node that you wish to upgrade first. It must have the `/etc
 
    ```shell
    # replace x in {{< skew currentVersion >}}.x-* with the latest patch version
-   apt-mark unhold kubeadm && \
-   apt-get update && apt-get install -y kubeadm='{{< skew currentVersion >}}.x-*' && \
-   apt-mark hold kubeadm
+   sudo apt-mark unhold kubeadm && \
+   sudo apt-get update && apt-get install -y kubeadm={{< skew currentPatchVersion >}} && \
```

Comment:
Does this work (without the `-*`)?

Reply:
I'm afraid the package version is something like `v1.28.1-0`, so the trailing `-0` cannot be omitted.

Reply:
So

```txt
kubeadm={{< skew currentPatchVersion >}}
```

Should be changed back to indicate that trailing packager revision, right?

```txt
kubeadm={{< skew currentPatchVersion >}}-*
```